graphql-api/src/graphql/schema.ts

```diff
@@ -25,6 +25,7 @@ import variantResolvers from './resolvers/variants'
 import variantFieldResolvers from './resolvers/variant-fields'
 import variantCooccurrenceResolvers from './resolvers/variant-cooccurrence'
 import cnvCoverageResolvers from './resolvers/cnv-coverage'
+import vaResolvers from './resolvers/va'
```

Comment:
I get a typescript error from here (as `./resolvers/va` doesn't have a default export). 

```
❯ pnpm typecheck

> @ typecheck /Users/rgrant/dev/work-broad/gnomad-browser
> pnpm tsc --noEmit

graphql-api/src/graphql/schema.ts:28:8 - error TS1192: Module '"/Users/rgrant/dev/work-broad/gnomad-browser/graphql-api/src/graphql/resolvers/va"' has no default export.

28 import vaResolvers from './resolvers/va'
          ~~~~~~~~~~~


Found 1 error in graphql-api/src/graphql/schema.ts:28

ELIFECYCLE  Command failed with exit code 2.
```

Is it possible that some refactoring happened and this error got mistakenly introduced? I was going to start a local API to check the VA fields that become queryable by this PR, and got this error.

---

As something not directly related:

I'm not sure why the `graphql-api` CI didn't run on this PR, the yaml file makes it look like it should have run.

```
...
 pull_request:
    paths:
      - 'graphql-api/**'
      - 'dataset-metadata/**'
      - 'package.json'
      - 'pnpm-lock.yaml'
      - '.github/workflows/**'
...
```

Reply:
I removed some unused code at the very end but missed a few references to it due to negelecting to run `typecheck`, fixed